lib/auth/trust/trustv1/service_test.go

```diff
@@ -742,6 +747,66 @@ func TestUpsertCertAuthority(t *testing.T) {
 	}
 }
 
+func TestRotateExternalCertAuthority(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	p := newTestPack(t)
+
+	authorizer := &fakeAuthorizer{
+		checker: &fakeChecker{
+			allow: map[check]bool{
+				{types.KindCertAuthority, types.VerbRotate}: true,
+			},
+		},
+	}
+
+	trust := local.NewCAService(p.mem)
+	cfg := &ServiceConfig{
+		Cache:      trust,
+		Backend:    trust,
+		Authorizer: authorizer,
+		AuthServer: &fakeAuthServer{
+			clusterName: &types.ClusterNameV2{
+				Spec: types.ClusterNameSpecV2{
+					ClusterName: "local",
+				},
+			},
+		},
+	}
+
+	service, err := NewService(cfg)
+	require.NoError(t, err)
+
+	localCA := newCertAuthority(t, types.HostCA, "local").(*types.CertAuthorityV2)
+	externalCA := newCertAuthority(t, types.HostCA, "external").(*types.CertAuthorityV2)
+	require.NoError(t, trust.UpsertCertAuthority(ctx, externalCA))
+
+	tests := []struct {
+		name    string
+		ca      *types.CertAuthorityV2
+		wantErr error
+	}{
+		{
+			name:    "NOK rotate local ca",
+			ca:      localCA,
+			wantErr: trace.BadParameter("can not rotate local certificate authority"),
```

Comment:
nit: Maybe, so it doesn't capture line info:

```suggestion
			wantErr: &trace.BadParameterError{Message: "can not rotate local certificate authority"},
```